specification/columns/commitmentdiscountquantity.md

```diff
@@ -10,17 +10,17 @@ The CommitmentDiscountQuantity column adheres to the following requirements:
 
 * CommitmentDiscountQuantity MUST be present in a FOCUS dataset when the provider supports *commitment discounts*.
```

Comment:
Sorry, this might have been lost in translation earlier ... 

1. Identify just the granular points which you feel are missing and then incorporate these into this pr
2. Incorporate these points into this pr, without effectively rewriting it.

Reply:
Discussed in TF-1 today, going to revisit a more wide-ranging readability topic in a later release